### PR TITLE
Segment Radial Distances added in features

### PR DESCRIPTION
**Pull request by eleftherioszisis**
Added the aforementioned feature that I used for the glial cell analysis. Sorry for the mess with the merge conflicts :P

### CODE REVIEW

File: neurom/features/_impl.py [hunk 1]
```diff
@@ -138,6 +137,24 @@ def section_path_distances(neurites, use_start_point=False, neurite_type=Neurite
...
+    '''Get an iterable containing section radial distances to origin of\
+       all neurites of a given type
+
+    Parameters:
+        origin: Point wrt which radial dirtance is calulated\
+                (default tree root)
+        use_start_point: if true, use the section's first point,\
+                         otherwise use the end-point (default False)
+        neurite_type: Type of neurites to be considered (default all)
+    '''
```
Comment (juanchopanza) [inline neurom/features/_impl.py]: Could you put the `i_segment_radial_dist` code in here? I really want to get away from these specialized iterator functions that don't anything useful. I want to remove these from morphtree, not add more.

**Comment by juanchopanza** [comment]
Could you squash these changes? 18 commits seems like a lot for one feature.

**Comment by eleftherioszisis** [comment]
So... how can I squash changes?

File: neurom/features/_impl.py (hunk 1 above)
Comment (eleftherioszisis) [inline neurom/features/_impl.py]: Sure.

**Comment by juanchopanza** [comment]
I found [this](http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit). I haven't tried it yet though. It might require that we close this pull request, you do some jiggling on your side because you already pushed the commits to your origin.

**Comment by eleftherioszisis** [comment]
tbh I am kinda lost... I tried the interactive rebase for my latest 8 commits but it throws an error that it cannot apply the segment radial distances commit. Maybe you should close the pull request?

**Comment by juanchopanza** [comment]
OK, I'll close it.

**Pull request closed without merging**